packages/lit-dev-content/site/tutorials/content/svg-templates/03.md

```diff
@@ -0,0 +1,98 @@
+### Overview
+
+This is chapter of the demo covers the basics of `<defs>` and `<use>` in
+SVG including how to:
+
+- define a group in defs
+- create a group of groups with `<use>`
+- create clip path
+- apply clip paths to a group
+
+### Learn
+
+The tile in this demo will be a series of transformed texts.
+
+Clip paths can be used to corral a group of visual elements. In SVG
```

Comment:
corral I don't feel like truly explains what it does. Maybe pull wording from MDN?

https://developer.mozilla.org/en-US/docs/Web/SVG/Element/clipPath

Reply:
done!